SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -12677,6 +12688,7 @@ CNSSolver::CNSSolver(void) : CEulerSolver() {
   Surface_CL_Visc = NULL; Surface_CD_Visc = NULL; Surface_CSF_Visc = NULL; Surface_CEff_Visc = NULL;
   Surface_CFx_Visc = NULL;   Surface_CFy_Visc = NULL;   Surface_CFz_Visc = NULL;
   Surface_CMx_Visc = NULL;   Surface_CMy_Visc = NULL;   Surface_CMz_Visc = NULL;
+  Surface_HF = NULL; Surface_MaxHF = NULL;
```

Comment:
Are `Surface_HF `and `Surface_MaxHF` used just for a viscous case? If so, maybe adding `_Visc` would help?